cve_bin_tool/version_scanner.py

```diff
@@ -334,6 +340,11 @@ def scan_and_or_extract_file(
 
     def recursive_scan(self, scan_path: str) -> Iterator[ScanInfo]:
         """Recursively scan files and directories, extracting information, and yielding the results using a generator."""
+        if self.no_scan:
+            LOGGER.info("No Scan Mode: No CVE Scanning")
+            LOGGER.info(
```

Comment:
I think we want to keep this pair as `info` but all the other log stuff should go to debug for now, just so if I manage to cut a release next week we don't make a bunch of surprising noise.